.github/workflows/beam_Metrics_Report.yml

```diff
@@ -90,11 +90,13 @@ jobs:
           name: Metrics Report
           path: "${{ github.workspace }}/.test-infra/jenkins/metrics_report/beam-metrics_report.html"
       - name: Generate Date
+        if: always()
```

Comment:
Why do we want to always send the result for this workflow, think its better to leave this one fail in background and fix it if needed because we would not like to spam dev with some incomplete result. 

Also we have to plan for changing the action parameters to really send out to dev instead to danny and me. Perhaps this PR can handle that

Reply:
https://github.com/apache/beam/pull/29136 The latter concern is solved by the pr here

Reply:
+1 on not adding the condition here and just letting this fail

Reply:
Updated